specs/binding-sse.spec/src/main/scripts/io/aklivity/zilla/specs/binding/sse/schema/sse.schema.patch.json

```diff
@@ -40,6 +40,36 @@
                             {
                                 "type": "integer",
                                 "default": 2000
+                            },
+                            "paths":
```

Comment:
Not sure about `paths` ... `path`, maybe `requests` ... `paths`, or `types` ... `path`?
Let's discuss the naming.

Reply:
Yeah, I was thinking about this as well a lot, but couldn't come up with a better one. I think the `requests` ... `path` makes more sense.

Reply:
Agreed, let's go with `requests` then, I was leaning towards that one too. 😄 

The idea of `types` is too overloaded already, it's why we went with `model` for the validation.